### DISCUSSION TITLE
能不能改为Autohotkey v2版？

### DISCUSSION THREAD

**Discussion by valuex** [zhugecaomao/ALTRun#7] [General]
趁现在代码规模量还不是很大，能不能用Autohotkey v2改写下。V2语法可读性更高，后面扩展也相对容易些。

**Comment by zhugecaomao** [comment]
> 趁现在代码规模量还不是很大，能不能用Autohotkey v2改写下。V2语法可读性更高，后面扩展也相对容易些。

谢谢你的建议，已经在尝试改成v2了，不过v1源代码还在精简中，改v2的进程会在精简之后继续
